spec/fernet_spec.rb

```diff
@@ -76,4 +76,33 @@
     expect(verifier.valid?).to eq(true)
     expect(verifier.message).to eq('password1')
   end
+
+  it 'verifies the token using one of several supplied keys' do
+    token = Fernet.generate(secret, 'harold@heroku.com')
+
+    verifier = Fernet.verifier(bad_secret, token, additional_secrets: [secret])
+
+    expect(verifier).to be_valid
+    expect(verifier.message).to eq('harold@heroku.com')
+  end
+
+  it 'ignores nil values as additional secrets' do
```

Comment:
For a library handling cryptography, I think a sensible default is to error on invalid parameters instead of ignoring them. It's easy enough to add `Array#compact` in the initialization code if you're reading environment variables that may be unset.